src/Week.jsx

```diff
@@ -97,6 +97,7 @@ export default class Week extends PureComponent {
           className={styles.header}
           style={{
             paddingLeft: RULER_WIDTH_IN_PIXELS,
+            paddingRight: '15px',
```

Comment:
It turns out this makes things look quite bad in chrome (at least on my machine). I'll see if I can come up a fix. 

<img width="584" alt="screen shot 2017-04-29 at 22 21 08" src="https://cloud.githubusercontent.com/assets/578277/25558594/2ecb421a-2d2a-11e7-81c7-f4e36a8ddac9.png">